### ISSUE TITLE
Not available on PyPI (contrary to README)

### ISSUE THREAD

**Issue by arahayrabedian** [evrom/wtforms-nocaptcha#1]
Hi @evrom,

Installation instructions state `pip install wtforms-nocaptcha` but there is no corresponding package on PyPi.

I'm going to give it a spin installing straight from git, but would be nice to see this (back?) on PyPI.

Thanks.

**Comment by evrom** [comment]
I don't think I put it on PyPI, and I will update the README.  I don't maintain this repository now, so if you use it, I recommend forking this repository for now!

**Comment by arahayrabedian** [comment]
yeah, figured it's unmaintained, trying to get it running on my fork.
